inc/template-functions.php

```diff
@@ -30,3 +30,31 @@ function ampconf_pingback_header() {
 	}
 }
 add_action( 'wp_head', 'ampconf_pingback_header' );
+
+/**
+ * Adds the search for as the last top-level menu item.
+ *
+ * @param string   $items The HTML list content for the menu items.
+ * @param stdClass $args  An object containing wp_nav_menu() arguments.
+ *
+ * @return string $items
+ */
+function ampconf_wp_nav_menu_items( $items, $args ) {
+	if ( 'header' !== $args->theme_location ) {
+		return $items;
+	}
+
+	$form = '<li class="menu-item menu-item-search-form">
+		<form role="search" method="get" target="_top" action="' . esc_url( home_url( '/' ) ) . '">
```

Comment:
@ThierryA the reason I did not use `get_search_form()` here is the default search for does not have the correct markup. We could create a `search-form.php` template with the markup required for this search form, but then that would apply to all other instances of `get_search_form()` as well.

Reply:
@kopepasah I think you should go ahead and use `get_search_form()`. The form sanitizer in https://github.com/Automattic/amp-wp/pull/871 will automatically add `target="_top"` to the `form` here. Otherwise, we could still use `get_search_form()` and then use the `get_search_form` filter to inject the `target` attribute onto it.

Reply:
@westonruter the reason for this markup is not only to add the `target` attribute, it was for stylistic purposes. However, if it is required, I can rework it to use `get_search_form()`.

Reply:
@kopepasah if just for stylistic purposes then I think it would be good to create a `search-form.php` and then also use `get_search_form()`, like you suggested. We'd be making sure that the AMP plugin forces the default output of  `get_search_form()` to be valid AMP, so it would be a good pattern to use for other themers to copy.

Reply:
I ended up creating a custom template and used `get_search_form()`.